docs/naps/8-telemetry.md

```diff
@@ -0,0 +1,251 @@
+:orphan: 
+  
```

Comment:
```suggestion  
```

Delete?

Reply:
This is present also in main repository. So maybe PR to docs repo and cherry pick such PR?

Reply:
Done, approved, and marked ready-to-merge:
https://github.com/napari/docs/pull/269
So merge and cherry-pick at your leisure